### PR TITLE
Add read-only in asset attribute and example fixes

### PR DESCRIPTION
**Pull request by NolwennP**

### CODE REVIEW

File: content/concepts/asset-manager.md [hunk 1]
```diff
@@ -1755,14 +1769,14 @@ And here is the JSON format of the `media_file` attribute type, for an image.
...
+  "is_read_only": false,
   "allowed_extensions": ["jpg"],
   "max_file_size": "10"
 }
 ```
 
 ::: info
-As of the v3.2 and v4.0, you can only have images in this attribute. Indeed, the `media_type` field only accepts the `image` value.  
-In the next version, you will be able to add new types of files (like PDFs).
+As of the v3.2 and v4.0, you can only have images in this attribute. Indeed, the `media_type` field accepts the `image`, `youtube`, `pdf` and `other` values.
```
Comment (NolwennP) [inline content/concepts/asset-manager.md]:
```suggestion
The `media_type` field accepts the `image` and `other` values.  
Starting from the v4, it also allows `pdf`.
```

File: content/concepts/asset-manager.md [hunk 2]
```diff
@@ -203,7 +213,7 @@ Not comfortable with regular expressions? You can try yours [right here](https:/
 
 Let's take an example to make this clearer! 
 ```regexp
-/(?P<product_ref>.*)-.*-(?P<attribute_ref>.*)\.jpg/
+/(?P<product_ref>.*)\\_(?P<attribute_ref>.*)\\.jpg/
```
Comment (mmetayer) [inline content/concepts/asset-manager.md]:
```suggestion
/^(?P<product_ref>.*)_(?P<attribute_ref>.*)\\.jpg$/
```

Comment (mmetayer) [reply content/concepts/asset-manager.md]: - no need to escape an underscore in a regexp
- I added the "beginning of string" (`^`) and "end of string" (`$`) anchors, which I believe will match most of the filenames

File: content/swagger/resources/asset_families/routes/asset_families.yaml [hunk 3]
```diff
@@ -51,12 +51,21 @@ get:
                 "en_US": "Packshots",
                 "fr_FR": "Plans produit"
               },
+              “naming_convention”:{
+                "source": {
+                  "property": "code",
+                  "channel": null,
+                  "locale": null
+                },
+                "pattern": "/?P(<product_ref>.*)\\.jpg/",
```
Comment (mmetayer) [inline content/swagger/resources/asset_families/routes/asset_families.yaml]:
```suggestion
                "pattern": "/(?P<product_ref>.*)\\.jpg/",
```

**Pull request merged**